### ISSUE TITLE
Interface Visibility

### ISSUE THREAD

**Issue by luthandobanda777-boop** [luthandobanda777-boop/Emergency-SOS-App#8]
Lack of accessibility options for visually impaired users

**Comment by luthandobanda777-boop** [comment]
Add voice commands and screen reader compatibility
